Cargo.toml

```diff
@@ -11,7 +11,7 @@ keywords      = ["tar", "s3", "archive"]
 license       = "Apache-2.0 OR MIT"
 readme        = "README.md"
 repository    = "https://github.com/elastio/ssstar"
-version       = "0.5.0"
+version       = "0.5.1"
```

Comment:
Updating the crate versions is part of the release process, not part of the PR.  Revert the changes to the crate versions.

Reply:
reverted. But I thought it already has to have version `0.5.1` because the `0.5.1` is already released but in master `ssstar` crate has version `0.5.0`

Reply:
If so that's an issue w/ the release process.  It shouldn't be patched around in this PR.  Let's revisit that when it's time to release this change.